sbb_newspapers_org_image/cli.py

```diff
@@ -0,0 +1,107 @@
+import click
+from sbb_newspapers_org_image.eynollah import eynollah
+
+
+@click.command()
+@click.option(
+    "--image", "-i", help="image filename", type=click.Path(exists=True, dir_okay=False)
+)
+@click.option(
+    "--out",
+    "-o",
+    help="directory to write output xml data",
+    type=click.Path(exists=True, file_okay=False),
+)
+@click.option(
+    "--model",
+    "-m",
+    help="directory of models",
+    type=click.Path(exists=True, file_okay=False),
+)
+@click.option(
+    "--save_images",
+    "-si",
+    help="if a directory is given, images in documents will be cropped and saved there",
+    type=click.Path(exists=True, file_okay=False),
+)
+@click.option(
+    "--save_layout",
+    "-sl",
+    help="if a directory is given, plot of layout will be saved there",
+    type=click.Path(exists=True, file_okay=False),
+)
+@click.option(
+    "--save_deskewed",
+    "-sd",
+    help="if a directory is given, deskewed image will be saved there",
+    type=click.Path(exists=True, file_okay=False),
+)
+@click.option(
+    "--save_all",
+    "-sa",
+    help="if a directory is given, all plots needed for documentation will be saved there",
+    type=click.Path(exists=True, file_okay=False),
+)
+@click.option(
+    "--allow_enhancement",
+    "-ae",
+    is_flag=True,
+    help="if this parameter set to true, this tool would check that input image need resizing and enhancement or not. If so output of resized and enhanced image and corresponding layout data will be written in out directory",
+)
+@click.option(
+    "--curved_line",
+    "-cl",
+    is_flag=True,
+    help="if this parameter set to true, this tool will try to return contoure of textlines instead of rectabgle bounding box of textline. This should be taken into account that with this option the tool need more time to do process.",
+)
+@click.option(
+    "--full_layout",
+    "-fl",
+    is_flag=True,
+    help="if this parameter set to true, this tool will try to return all elements of layout.",
```

Comment:
It wasn't, I fixed this in https://github.com/qurator-spk/eynollah/pull/2/commits/97bc57be35a21f2080542c9bae8b963da8fabdb3 @vahidrezanezhad you can define command line options as "flags", so they represent boolean values which makes the code cleaner because you don't need to compare strings `"true"`, `"TRUE"` etc. if defined like this.